runtime/suggestion.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
I hope that I understood you correctly, if not, could you, please, provide more detailed guidance to the changes you are recommending. Thanks!